example/styles/globals.css

```diff
@@ -17,13 +16,22 @@ body::before {
   bottom: 0;
   left: 0;
   background: url('/background.svg') no-repeat;
-  background-position: 0px 250px;
+  /* background-position: 0px 250px; */
   width: 100%;
   height: 100%;
   background-size: cover;
   opacity: 0.5;
 }
-
+input {
+  
+  font-family: Inconsolata, monospace, sans-serif!important;
```

Comment:
One time this instructios are being addes as well on `_app.tsx` are they needed here?